hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/pipelines/pipelines.tsx

```diff
@@ -247,7 +247,7 @@ export class Pipelines extends React.Component<Record<string, object>, IPipeline
                 }, [])}
                 loading={activeLoading} pagination={paginationConfig} rowKey='pipelineId'/>
             </TabPane>
-            <TabPane key='2' tab='Inactive'/>
+
```

Comment:
hmm. Wouldn't a proper solution be **adding** inactive pipeline instead? Have we checked how much effort this would be to by modifying the [existing](https://github.com/apache/ozone/blob/ab608c5d4649b2b320e732b042998bf6ece77cc9/hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/PipelineEndpoint.java#L72) endpoint (and possibly some OM code as well)?

Reply:
I found this relevant TODO:

https://github.com/apache/ozone/blob/a02c6df497b2a636c7a81c26e53f39daa7958841/hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconPipelineManager.java#L132-L134

Reply:
@smengcl, thanks for taking a look at this change. The TODO referred in [ReconPipelineManager.java](https://github.com/apache/ozone/blob/a02c6df497b2a636c7a81c26e53f39daa7958841/hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconPipelineManager.java#L132-L134) talks about tracking pipelines which are closed and deleted in SCM. In Recon, it is possible to maintain the history of pipelines and show it to the user. This functionality can be easily added. But calling it an "Inactive" pipeline will bring in confusion as we have something called Dormant/Inactive state for a pipeline in SCM, which was introduced for debugging purposes. We also have _activate_ and _deactivate_ pipeline commands in SCM.
Ref: [HDDS-1810](https://issues.apache.org/jira/browse/HDDS-1810)

To have clarity, we can have a different tab name (something other than Inactive) in Recon to show the Pipeline history (deleted pipelines). We can do this when we start working on that functionality.


Since "activate" and "deactivate" (Dormant State) was introduced for testing/debugging purposes, it is better to hide them from the end user in the Recon UI.

Reply:
Okay. If this is the case, shall we hide the "Active" tab label as well?

Reply:
Yeah, that makes sense. We can hide the "Active" tab as well till we work on displaying closed/deleted pipeline history.

Reply:
Sounds good @nandakumar131 . Simply hiding the "Active" label from the end user should be good enough (for now) if removing the whole `<div>` is a big change.

Reply:
@smengcl @nandakumar131 , this has been taken care, pls re-review and merge.